Fluid.Tests/StringFiltersTests.cs

```diff
@@ -199,7 +199,7 @@ public void Truncate()
         {
             var input = new StringValue("Hello World");
 
-            var arguments = new FilterArguments().Add(new NumberValue(5)).Add(new StringValue("..."));
+            var arguments = new FilterArguments().Add(new NumberValue(8));
```

Comment:
Can you refactor to a `[Theory]` with `input`, `output` and `size` to handle the edge cases?